components/nimbus/src/targeting.rs

```diff
@@ -61,6 +61,26 @@ impl NimbusTargetingHelper {
     }
 }
 
+impl From<TargetingAttributes> for NimbusTargetingHelper {
+    fn from(value: TargetingAttributes) -> Self {
+        cfg_if::cfg_if! {
+            if #[cfg(feature = "stateful")] {
+                let store = Arc::new(Mutex::new(EventStore::new()));
+                NimbusTargetingHelper::new(value, store)
+            } else {
+                NimbusTargetingHelper::new(value)
+            }
+        }
+    }
+}
+
+impl From<AppContext> for NimbusTargetingHelper {
+    fn from(context: AppContext) -> Self {
+        let ta: TargetingAttributes = context.into();
+        ta.into()
+    }
+}
+
```

Comment:
If you're just using `TargetingHelper::new()` in [`cirrus_client.rs`](https://github.com/mozilla/application-services/pull/5471/files#diff-f13145056c9a37d6aec2871fb99b16ebd1340212a3354e1dfee5a4bdb4219332R97) then this can stay where it is.